Dockerfile.template

```diff
@@ -71,8 +71,21 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
 	&& \
 	# we can change from world to world-bin in newer releases
 	make world && \
-	make install-world
-
+	make install-world && \
+	# clones pg_cron extension for local use, this is not available wihout installing
+ 	git clone https://github.com/citusdata/pg_cron.git /pg_cron && \
+	cd pg_cron && make && make install && \
+	cron_conf=/var/lib/postgresql/data/custom-conf.conf && echo "" > $cron_conf && \
+	echo "shared_preload_libraries = 'pg_cron'" >> $cron_conf && \
```

Comment:
now that the `postgres.conf` file has been merged in, do you want to move some of these changes over to that file or make a separate one for the custom-cron.conf? something like:
```
shared_preload_libraries = 'pg_cron'
cron.database_name = 'postgres'
```
and 
`include = '/var/lib/postgresql/data/custom-conf.conf'` in `postgres.conf`